.github/workflows/lint.yaml

```diff
@@ -11,6 +11,10 @@ on:
   pull_request:
     branches: [ "main", "release-2.5" ]
 
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
```

Comment:
@bestbeforetoday 
I came to the same conclusion as you about canceling existing runs when a PR is pushed again, but implemented it with a little different `group` specification relative to yours at:
https://github.com/hyperledger/fabric-gateway/blob/b1659c8d92917b3237aaceb9f6d039c049c6897a/.github/workflows/pull_request.yml#L8-L10

github.head_ref will be the user's branch name, so I think the end result is the same... do you have a preference on the approach?

Reply:
It looks like the fabric-samples build kicks of multiple workflows in response to a PR. To avoid one PR workflow cancelling other workflows for the same PR, you might need to include the `github.workflow` as part of the group name, so each workflow's group is unique. This example from the GitHub docs is essentially the same as the approach I took, just always using the ref rather than using the PR number where present:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow